### PR TITLE
Updated PR template taking wally into account.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

**Comment by SirVer** [comment]
@damonkohler Could you review for English? @cschuet Please review for content.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-Want to contribute? Great! Make sure you've checked all these boxes before creating your PR:
+Want to contribute? Great! Make sure you've read, understood and considered all
```
Comment (damonkohler) [inline .github/PULL_REQUEST_TEMPLATE.md]: One sentence per line.

Comment (SirVer) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ah, I leave this to @cschuet.

This is never displayed as Markdown, it is only visible as pure text to our contributors in the issue text. For the README.md, Christoph preferred the readability of block-text over the diffability of one line per sentence.

**Comment by SirVer** [comment]
@cschuet ping?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (cschuet) [reply .github/PULL_REQUEST_TEMPLATE.md]: The commit message box does not resize and displays the raw text and not the MD as @SirVer mentioned, so I would prefer block-text for better readability.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**